### ISSUE TITLE
Valid feedback in form validation are not accessible for assistive technologies

### ISSUE THREAD

**Issue by MewenLeHo** [Orange-OpenSource/Orange-Boosted-Bootstrap#1599]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

In [form validation](https://boosted.orange.com/docs/5.2/forms/validation/#custom-styles), when using:

`<div class="valid-feedback">Looks good!</div>`

The feedback is visually hidden but still accessible to ATs users thanks to:

```
.valid-feedback, .visually-hidden, .visually-hidden-focusable:not(:focus):not(:focus-within) {
    position: absolute!important;
    width: 1px!important;
    height: 1px!important;
    padding: 0!important;
    margin: -1px!important;
    overflow: hidden!important;
    clip: rect(0,0,0,0)!important;
    white-space: nowrap!important;
    border: 0!important;
}
```

But there is also a

```
.valid-feedback {
    display: none;
}
```

rule coming from:

```
  @if $state == "valid" {
    .#{$state}-feedback {
      display: none;
      @extend .visually-hidden; // Boosted mod: Do not visually display .valid-feedback
    }
```
in `scss\mixins\_forms.scss`

The problem is that using `display: none;` will also prevent it to be rendered by the ATs.

When a field is valid, we only have:
![screenshot-boosted orange com-2022 11 08-10_51_31](https://user-images.githubusercontent.com/92363071/200533253-e2ad3f2f-0488-4d9c-87f6-1fe9127b438a.png)

So we have a visual feedback but it's not accessible for ATs.
It means actually when using Boosted, ATs can't _explicitly_ stated that a field is valid.

### Motivation and context

The good part is that, if we apply some actions spotted during the DevTestDays preparation and especially [In Docs / Forms / Validation, No programmatic link between error messages dans input fields #37405](https://github.com/twbs/bootstrap/issues/37405), a valid field can be  _implicytly_ stated as valid because of not being  _explicitly_ stated as invalid.

=> for a valid preview, see [https://deploy-preview-1559--boosted.netlify.app/docs/5.2/examples/form/](https://deploy-preview-1559--boosted.netlify.app/docs/5.2/examples/form/) in [Add form example in boosted doc #1559](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1559)

The question is what should we do while waiting the opinion of Boostrap regarding the issue opened by @Aniort? 

Some options:
Made the changes only in Boosted?
Remove the `display: none;` rule?
Start using `aria-invalid="<true | false>"` in Boosted forms to provide a feedback?

**Comment by MaxLardenois** [comment]
Boosted only, won't fix
